### PR TITLE
Remove validation from method of advice question

### PR DESCRIPTION
**Pull request by alexwllms**
@jongilbraith @lshepstone

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -203,7 +203,7 @@ en:
       label_five: Post code *
 
     in_person_advice_methods:
-      heading: Does your firm offer advice (through to transaction) in person? *
```
Comment (lshepstone) [inline config/locales/en.yml]: There might also be an entry that's not longer needed at the top of this file.

```
  in_person_advice_methods:
    too_short: "- please select at least one"
```

Comment (benlovell) [inline config/locales/en.yml]: Good catch. Will remove.

**Comment by lshepstone** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1: :ship:

**Pull request merged**